helm/karpenter-crossplane-resources/values.yaml

```diff
@@ -13,3 +13,7 @@ awsCluster:
 project:
   branch: "[[ .Branch ]]"
   commit: "[[ .SHA ]]"
+
+# internal:
```

Comment:
Leftover? :slightly_smiling_face:

Reply:
Only our cluster apps have `make normalize-schema generate-values`, so this is manual and I guess okay to comment out

Reply:
yes I added it just as an example. But since it's an optional value I left it commented out.